src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import TodoContainer from './components/TodoContainer';
+import Navbar from './components/Navbar';
+import About from './components/About';
+import Contact from './components/Contact';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
You initialized the constructor with props and state here but did not use it in your component, I'd suggest you remove them since you are not using them within your components. This will ensure you do not have redundant/unused code within your codebase.